packages/back-end/src/services/auth/index.ts

```diff
@@ -248,6 +254,45 @@ export async function processJWT(
         dateCreated: new Date(),
       });
     };
+
+    if (IS_CLOUD) {
```

Comment:
Also we likely need to have another flag if we want to force this for dev in localhost. For now I can just comment this out when testing.